compiler/luci/logex/CMakeLists.txt

```diff
@@ -19,6 +21,12 @@ install(TARGETS luci_logex DESTINATION lib)
 install(DIRECTORY include/ DESTINATION include
         FILES_MATCHING PATTERN "*.h")
 
+add_executable(luci_input_names ${SOURCES} ${INPUT_NAMES_SRC})
```

Comment:
`luci` provides libs where `luci_input_names` resides inside `luci/logex` which looks a bit strange.

what I thought in mind is to introduce `compiler/circle-input-names` tool that uses `luci/logex`. we may have to relocate `CircleNodeSummaryBuilders.h` to `include` folder.

Reply:
Thanks for reviewing, I'll revise it by referring to your comment.